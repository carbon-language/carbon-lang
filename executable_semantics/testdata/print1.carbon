// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
// RUN: ../executable_semantics %s | \
// RUN:   FileCheck %s

package ExecutableSemanticsTest api;

fn main() -> i32 {
  var s: auto = "Hello world!\n";
  Print(s);
  return 0;
}

// AUTOUPDATE: ../executable_semantics %s
// CHECK: Hello world!
// CHECK: result: 0
