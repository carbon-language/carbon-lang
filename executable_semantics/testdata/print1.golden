Hello world!
result: 0
